src/test/java/org/junit/tests/experimental/results/ResultMatchersTest.java

```diff
@@ -20,4 +29,29 @@ public void hasFailuresDescriptionReflectsInput(int i) {
         assertThat(ResultMatchers.failureCountIs(i).toString(),
                 containsString("" + i));
     }
+
+    @Test
+    public void testFailureCountIsTakingMatcher_givenResultWithOneFailure() {
+        PrintableResult resultWithOneFailure = new PrintableResult(Collections.singletonList(
+                new Failure(Description.createTestDescription("class", "name"), new RuntimeException("failure 1"))));
+
+        assertThat(ResultMatchers.failureCountIs(equalTo(3)).matches(resultWithOneFailure), is(false));
+        assertThat(ResultMatchers.failureCountIs(equalTo(1)).matches(resultWithOneFailure), is(true));
+    }
+
+    @Test
+    public void failureCountIsTakingMatcherShouldHaveGoodDescription() {
+        Matcher<Integer> matcherTestStub = new BaseMatcher<Integer>() {
```

Comment:
What's wrong?
Compilation looks fine to me:
https://travis-ci.org/junit-team/junit4/builds/131156153